### PR TITLE
Adding new press clip

### PR DESCRIPTION
**Pull request by LilyLME**
submitting to add a new press clip as per this [issue](https://dip.torproject.org/torproject/web/tpo/issues/53)

### CODE REVIEW

File: content/press/36-civil-rights-groups-demand-end-to-amazons-partnerships-with-police/contents.lr [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+---
+active: True
+---
+type: snippet
+---
+publisher:  VICE
+---
+title:  36 Civil Rights Groups Demand End to Amazon’s Partnerships with Police
+---
+link:  https://www.vice.com/en_us/article/ne8q3w/36-civil-rights-groups-demand-end-to-amazons-partnerships-with-police?utm_source=mbtwitter
```
Comment (emmapeel2) [inline content/press/36-civil-rights-groups-demand-end-to-amazons-partnerships-with-police/contents.lr]: i think we could take the twitter suffix away, '?utm_source=mbtwitter'

File: content/press/the-watchdogs-tracking-secret-online-censorship-across-the-globe/contents.lr [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+active: True
+---
+type: snippet
+---
+publisher:  CNet
+---
+title:  These watchdogs track secret online censorship across the globe
+---
+link:  https://www.cnet.com/features/the-watchdogs-tracking-secret-online-censorship-across-the-globe-ooni/?ftag=COS-05-10aaa0b
```
Comment (emmapeel2) [inline content/press/the-watchdogs-tracking-secret-online-censorship-across-the-globe/contents.lr]: same here: '?ftag=COS-05-10aaa0b' is not needed for the url

File: content/press/36-civil-rights-groups-demand-end-to-amazons-partnerships-with-police/contents.lr (hunk 1 above)
Comment (LilyLME) [reply content/press/36-civil-rights-groups-demand-end-to-amazons-partnerships-with-police/contents.lr]: Thanks :), all done @emmapeel2

**Comment by emmapeel2** [comment]
merged! not sure why it looks like not...

**Comment by LilyLME** [comment]
It says torproject-pusher closed this pull request

**Comment by emmapeel2** [comment]
yeah, closed, but i thought it will appear merged... <confession>i squashed your commits </confession>

**Comment by LilyLME** [comment]
Oh okay, have no idea what's wrong. Incase you find out before me :), I'm here.

**Comment by LilyLME** [comment]
@emmapeel2, just saw the changes on the page. Super excited!! Thank you

**Pull request closed without merging**